api/api.go

```diff
@@ -0,0 +1,77 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/NebulousLabs/skynet-accounts/database"
+
+	"github.com/NebulousLabs/skynet-accounts/build"
+
+	"github.com/julienschmidt/httprouter"
+)
+
+// API is ...
+type API struct {
+	DB     *database.DB
```

Comment:
API doesn't need to expose its fields right? If not they should be staticDB and staticRouter according to Sia conventions.

Reply:
Done.